cirq/ops/three_qubit_gates.py

```diff
@@ -145,6 +149,132 @@ def __str__(self) -> str:
         return 'CCZ**{}'.format(self._exponent)
 
 
+@value.value_equality()
+class ThreeQubitDiagonalGate(gate_features.ThreeQubitGate):
+    """A gate given by a diagonal 8x8 matrix.
+
+    The list of angles passed to the constructor should be in radians.
```

Comment:
Move this into the `Args:` section of the docstring of `__init__`

Reply:
Done!